eva-pipeline/src/test/java/embl/ebi/variation/eva/pipeline/steps/VariantAnnotLoadBatchTest.java

```diff
@@ -24,17 +47,127 @@
  */
 public class VariantAnnotLoadBatchTest {
 
+
+    private VariantAnnotLoadBatch variantAnnotLoadBatch;
+    private ExecutionContext executionContext;
+
+    @Before
+    public void setUp() throws Exception {
+        executionContext = MetaDataInstanceFactory.createStepExecution().getExecutionContext();
+
+        variantAnnotLoadBatch = new VariantAnnotLoadBatch();
+    }
+
+    @Test
```

Comment:
This should be moved in a dedicated test for VariantAnnotationLineMapper. The test is for annotation load step, the check if the fields are parsed correctly should be done somewhere else.